hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/virtual/VirtualMapFactory.java

```diff
@@ -58,7 +58,7 @@ public class VirtualMapFactory {
     private static final long MAX_ACCOUNTS = 100_000_000L;
     private static final long MAX_TOKEN_RELS = 100_000_000L;
     private static final long MAX_SCHEDULE_SECONDS = 500_000_000;
-    private static final long MAX_IN_MEMORY_INTERNAL_HASHES = 0;
+    private static final long MAX_IN_MEMORY_HASHES = 0;
```

Comment:
Oh, this is interesting. I know it isn't a part of this fix, but why is hashes RAM/disk threshold is set to 0? It has huge impact on performance... May need to check with @tinker-michaelj about it

Reply:
I asked Michael, and it looks like this parameters was never given proper consideration. We have to find the right balance between memory consumption and hash look up performance. I think it's outside of the scope of this PR, so I filed #6043 to address that.